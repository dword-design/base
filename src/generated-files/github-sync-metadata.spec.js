import proxyquire from '@dword-design/proxyquire'

export default {
  'do not sync keywords': function () {
    const self = proxyquire('./github-sync-metadata', {
      '../config': {
        syncMetadata: false,
      },
    })
    expect(self).toMatchSnapshot(this)
  },
  valid() {
    const self = proxyquire('./github-sync-metadata', {
      '../config': {},
    })
    expect(self).toMatchSnapshot(this)
  },
}
