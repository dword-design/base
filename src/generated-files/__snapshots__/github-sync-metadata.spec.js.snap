// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`github-sync-metadata do not sync keywords 1`] = `
Object {
  "jobs": Object {
    "build": Object {
      "runs-on": "ubuntu-latest",
      "steps": Array [
        Object {
          "uses": "actions/checkout@v2",
        },
        Object {
          "uses": "dword-design/action-sync-node-meta@fork",
          "with": Object {
            "approve": false,
            "commitMessagePrefix": "fix:",
            "githubToken": "\${{ secrets.GITHUB_TOKEN }}",
            "syncKeywords": false,
          },
        },
      ],
    },
  },
  "name": "sync-metadata",
  "on": Object {
    "schedule": Array [
      Object {
        "cron": "0 5 * * *",
      },
    ],
    "workflow_dispatch": Object {},
  },
}
`;

exports[`github-sync-metadata valid 1`] = `
Object {
  "jobs": Object {
    "build": Object {
      "runs-on": "ubuntu-latest",
      "steps": Array [
        Object {
          "uses": "actions/checkout@v2",
        },
        Object {
          "uses": "dword-design/action-sync-node-meta@fork",
          "with": Object {
            "approve": false,
            "commitMessagePrefix": "fix:",
            "githubToken": "\${{ secrets.GITHUB_TOKEN }}",
            "syncKeywords": false,
          },
        },
      ],
    },
  },
  "name": "sync-metadata",
  "on": Object {
    "schedule": Array [
      Object {
        "cron": "0 5 * * *",
      },
    ],
    "workflow_dispatch": Object {},
  },
}
`;
